gnomad/sample_qc/relatedness.py

```diff
@@ -262,8 +268,13 @@ def infer_families(
     relationship_col: str = "relationship",
 ) -> hl.Pedigree:
     """
-    This function takes a hail Table with a row for each pair of individuals i,j in the data that are related (it's OK to have unrelated samples too).
-    The `relationship_col` should be a column specifying the relationship between each two samples as defined in this module's constants.
+    Generate a pedigree containing trios inferred from the `relationship_ht`.
+
+    This function takes a hail Table with a row for each pair of individuals i,j in the data that are
+    related (it's OK to have unrelated samples too).
```

Comment:
Maybe "each pair of related individuals i, j in the data" vs "each pair of individuals i,j in the data that are related"?